tests/config/base/openstack_control_plane.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  secret: osp-secret
+  storageClass: local-storage
+
+  cinder:
+    enabled: false
+    template:
+      cinderAPI: {}
+      cinderScheduler: {}
+      cinderBackup: {}
+      cinderVolumes: {}
+
+  dns:
+    enabled: true
+    template:
+      externalEndpoints:
+      - ipAddressPool: ctlplane
+        loadBalancerIPs:
+        - 192.168.122.80
+      options:
+      - key: server
+        values:
+        - 192.168.122.1
+      replicas: 1
+
+  glance:
+    enabled: false
+    template:
+      glanceAPIInternal: {}
+      glanceAPIExternal: {}
+
+  horizon:
+    enabled: false
+    template: {}
+
+  ironic:
+    enabled: false
+    template:
+      ironicConductors: []
+
+  keystone:
+    enabled: false
+    template: {}
+
+  manila:
+    enabled: false
+    template:
+      manilaAPI: {}
+      manilaScheduler: {}
+      manilaShares: {}
+
+  mariadb:
+    enabled: true
+    templates:
+      openstack:
+        storageRequest: 500M
+
+  memcached:
+    enabled: true
+
+  neutron:
+    enabled: false
+    template: {}
+
+  nova:
+    enabled: false
+    template: {}
+
+  ovn:
+    enabled: false
+    template:
+      ovnController:
+        external-ids:
+          system-id: "random"
+          ovn-bridge: "br-int"
+          ovn-encap-type: "geneve"
+      ovnDBCluster:
+        ovndbcluster-nb:
+          containerImage: quay.io/podified-antelope-centos9/openstack-ovn-nb-db-server:current-podified
```

Comment:
Do we need to mention the image here? I was looking at the code and looks like this is the default

https://github.com/openstack-k8s-operators/ovn-operator/blob/7a30c60d8624755159c5386e3078aa811a8f540c/config/default/manager_default_images.yaml#L15

Reply:
right @Sandeepyadav93 ... mainly I was trying to keep what was added very recently i.e. see the diff there https://github.com/openstack-k8s-operators/data-plane-adoption/pull/63/files#diff-99bc82f32e98ef131cd9bdf8c3840aca9a0cf58ef6620bac4abd594395991160 (I am basically just moving that from the ovn role to this base definition). I was going to catch up with @booxter about that this week and see if he really wants to specify that explicitly for some reason.

Reply:
thanks Marios for your comment, +1 from my side we can drop the explicit mention in a follow-up if this is not really needed.